### DISCUSSION TITLE
Is there a way to show icons on the Menubar Label?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [NativePHP/laravel#254] [Q&A] [answered]
Is it possible to add a color icon to the menubar. I am currently showing some stats on the menubar, but it would be great if there was a way to add icons. Any advice if it would be possible?
![CleanShot 2024-02-26 at 19 47 17](https://github.com/NativePHP/laravel/assets/1639302/ed688751-c1d6-43e6-b48a-8432a0d57976)

**Comment by simonhamp** [comment, marked as answer]
See #258
